raiden_contracts/tests/test_service_registry.py

```diff
@@ -303,6 +304,35 @@ def test_deprecation_switch(
         service_registry.functions.deposit(SERVICE_DEPOSIT).call_and_transact({"from": A})
 
 
+def test_depractaion_immeidate_payout(
```

Comment:
typo alarm, in the second and third word

Reply:
I think I made the same in the branch name.